pytorchfi/weight_error_models.py

```diff
@@ -0,0 +1,58 @@
+"""pytorchfi.error_models provides different error models out-of-the-box for use."""
+
+import logging
+import torch
+from pytorchfi import core
+from pytorchfi.util import *
```

Comment:
I think if we can import from line above to avoid `import *` that would be better practice

Reply:
good point.
Just wondering were should we `import random`? Should we do it in `util`, or in each separate file? And if in `util`, is there a clean way to import random without `import *`?

Reply:
I think we can import in each separate file, similar to what we have for `logging` and other packages.